src/test/java/com/wikia/webdriver/testcases/fandom/FanFeedTest.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.testcases.fandom;
+
+import com.wikia.webdriver.elements.fandom.components.FanFeed;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.elements.fandom.components.FanFeed'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)